Gemfile

```diff
@@ -1,6 +1,4 @@
 source 'https://rubygems.org'
-
-git_source(:github) { |repo_name| "https://github.com/#{repo_name}" }
+gem 'rubocop', '>= 1.0', '< 2.0'
 
 # gem "rails"
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you add some more details in your pull request summary to make your pull request professional :+1: Your pull request should have all the necessary details of what you are doing in this pull request :blush: